File: Python/motorcycle.py
from vehicle import Vehicle

class Motorcycle(Vehicle):
    def __init__(self, plate_number, brand, year, color, motor_type, tank_capacity):
        super().__init__(plate_number, brand, year, color)
        self.motor_type = motor_type
        self.tank_capacity = tank_capacity

    def display_info(self):
        super().display_info()
        print("Type:", self.motor_type)
        print("Tank Capacity:", self.tank_capacity, "liters")

File: CPP/motorcycle.cpp
#include <iostream>
#include <vector>
#include <string>
#include "Vehicle.cpp"
using namespace std;

class Motorcycle : public Vehicle {
private:
    string type;
    int tankCapacity;
public:
    Motorcycle(string plateNumber, string brand, int year, string color, string type, int tankCapacity) : Vehicle(plateNumber, brand, year, color), type(type), tankCapacity(tankCapacity) {}

    void displayInfo() override {
        Vehicle::displayInfo();
        cout << "Type: " << type << endl;
        cout << "Tank Capacity: " << tankCapacity << " liters" << endl;
    }
};